### DISCUSSION TITLE
Help getting small regression fix merged

### DISCUSSION THREAD

**Discussion by davecardwell** [axios/axios#6491] [General]
The 1.x line introduced a regression where the `responseDetails.headers` option to `beforeRedirect()` was dropped, along with the entire third `requestDetails` parameter.

A fix for `responseDetails.headers` was accepted in #5778, but I’m having trouble getting #6241 merged to fix the `requestDetails` parameter.

Since this was a regression I think #6241 should be an uncontroversial fix, and the PR has type updates and tests.

Is there anything I can do to expedite getting this merged? We’ve come to rely on this parameter being passed and it’s blocking our ability to upgrade to newer versions of Axios.

**Comment by jasonsaayman** [comment]
Closing as I think this is already resolved.
